pkg/controller/installation/core_controller.go

```diff
@@ -226,16 +227,21 @@ func GetInstallation(ctx context.Context, client client.Client, provider operato
 }
 
 // GetInstallation returns the default installation instance with defaults populated.
-func getInstallation(ctx context.Context, client client.Client, provider operator.Provider) (*operator.Installation, error) {
```

Comment:
Surely there must be a way to create a single clientset from these two individual ones?